src/librustc_mir/interpret/eval_context.rs

```diff
@@ -269,7 +280,9 @@ impl<'c, 'b, 'a, 'mir, 'tcx, M: Machine<'mir, 'tcx>> HasDataLayout
     }
 }
 
-impl<'a, 'mir, 'tcx, M: Machine<'mir, 'tcx>> layout::HasTyCtxt<'tcx> for &'a EvalContext<'a, 'mir, 'tcx, M> {
+impl<'a, 'mir, 'tcx, M> layout::HasTyCtxt<'tcx> for &'a EvalContext<'a, 'mir, 'tcx, M>
+where M: Machine<'mir, 'tcx>
```

Comment:
this should be indented? is this the best rearrangement of this?

Reply:
Not sure what our usual style is there, but yeah indentation makes sense.